web-server/src/components/Teams/useTeamsConfig.tsx

```diff
@@ -498,26 +499,26 @@ const useReposSearch = () => {
     async (query) => {
       depFn(isLoading.true);
       // cancel the previous request if it exists
-      if (cancelTokenSource || !query) {
-        cancelTokenSource.cancel('Operation canceled due to new request.');
+      if (controllerRef.current) {
+          controllerRef.current.abort('Operation canceled due to new request.');
       }
       if (!query) return depFn(isLoading.false);
       // create a new cancel token
-      cancelTokenSource = axios.CancelToken.source();
+      controllerRef.current = new AbortController();
 
       try {
         const response = await axios(
           `/api/internal/${orgId}/git_provider_org`,
           {
             params: { providers: integrationList, search_text: query },
-            cancelToken: cancelTokenSource.token
+            signal: controllerRef.current.signal
           }
         );
         const data = response.data;
         depFn(searchResults.set, data);
         depFn(isLoading.false);
       } catch (error: any) {
-        if (!axios.isCancel(error)) {
+        if (error.name !== 'CanceledError') {
```

Comment:
Possible to use instanceof CanceledError instead?

Reply:
Yes, sure it would work, I updated to **instanceof CanceledError** in this PR